aa_engine/aas_copyfromremote.m

```diff
@@ -100,10 +102,25 @@
     retrydelay=[1 2 4 8 16 32 64 128 256 512 768 1024 2048 1];
     for retry=retrydelay
```

Comment:
@tiborauer I noticed the exponential back-off here causes unnecessarily extra calls to the copying process solely because it is looped over and over again. It assumes the host machine is remote. However, for local connections it would not be necessary. 

I haven't used any remote connections, but even there I can't see a reason to have it this way because even if the connection is not refused the loop will be called over and over again.

Either we can re-factor the code completely, or just set the retrydelay array to something smaller for local pipelines.

Reply:
.... or we can move the `if isempty(host)` conditional block part out of the loop and only loop when it is not empty.

Reply:
Good idea!